cmd/config/internal/commands/cmdcreatesubstitution_test.go

```diff
@@ -184,6 +184,108 @@ spec:
         image: something/nginx::1.7.9/nginxotherthing # {"$openapi":"my-image-subst"}
       - name: sidecar
         image: sidecar:1.7.9
+ `,
+		},
+		{
+			name: "nested substitution",
+			args: []string{
+				"my-nested-subst", "--field-value", "something/nginx::1.7.9/nginxotherthing", "--pattern", "something/${my-image-subst}/${my-other-setter}"},
```

Comment:
Put this on multiple lines to make it easier to read.

Reply:
Done